### ISSUE TITLE
nohang and zfs should not be put on the same boat

### ISSUE THREAD

**Issue by liyimeng** [kairos-io/kairos#2258]
**Kairos version:**

2.5.0+
**CPU architecture, OS, and Version:**

NA
**Describe the bug**

When testing out kairos, I have [this issue](https://github.com/kairos-io/kairos/discussions/2240#discussioncomment-8478623
), I run a quick research and see [this](https://github.com/hakavlad/nohang?tab=readme-ov-file#known-problems), which say nohang are not running nicely with zfs [more info](https://github.com/openzfs/zfs/issues/10255)

Basically, it is a bad idea to have nohang and zfs in the same system since they are not playing nice to each other. 

Shall we remove nohang for now?

**To Reproduce**

**Expected behavior**

**Logs**

**Additional context**

**Comment by mauromorales** [comment]
Closing since there's already an issue filed https://github.com/kairos-io/kairos/issues/2195

Feel free to add any new info there, or re-open if you think they are different issues

**Comment by liyimeng** [comment]
Indeed it is likely the same issue, upstream have a fix for this https://github.com/hakavlad/nohang/issues/135,  hope new release of nohang fix the issue.
